configure.ac

```diff
@@ -2,7 +2,7 @@ AC_PREREQ(2.52)
 
 m4_define([PKG_VERSION_MAJOR], [3])
 m4_define([PKG_VERSION_MINOR], [6])
-m4_define([PKG_VERSION_PATCH], [9])
```

Comment:
I guess this should be, at the very least, a minor (probably a major version update)

Reply:
Looks like there is no ABI break. Unfortunately we do not use semantic versioning. Our ABI is `major.minor`, so bumping the minor version is quite a bother for packaging.
So as much as I agree it feels like a larger bump, in practice this looks like just a patch version bump.

Reply:
That's fine. Just let wanted to point it out

Reply:
All prior commands would continue working as-is. And I've had more or less this exact conversation with Tino before, and the next major bump will probably be #119 (if I ever finish it).

Reply:
👍